tests/e2e/src/pages/metamask.page.ts

```diff
@@ -209,7 +209,10 @@ export class MetamaskPage extends BasePage {
   async switchNetwork() {
     const switchNetworkBtnSelector = "//div[@class='transaction-footer-row']//button";
     const switchNetworkBtnElement: any = await this.world.page?.locator(switchNetworkBtnSelector);
-    if (await switchNetworkBtnElement.isEnabled()) {
+    //check that switchNetworkBtnSelector is switch network button
+    const buttonText = await switchNetworkBtnElement.innerText();
```

Comment:
This new if will check whether we have the button - Switch network. 
If not - the check will pass and make no changes (scenario - when the network is already correct).
Because await switchNetworkBtnElement.isEnabled() can only check whether the button is enabled. But it can be either Continue - or Switch network. Both option will give True, BUT Continue - won't trigger MM popup.